addons/editor-messages/userscript.js

```diff
@@ -9,9 +9,9 @@ export default async function ({ addon, console, msg }) {
   messageCount.classList.add("sa-editormessages-count");
   messages.appendChild(messageCount);
   const setMessages = async () => {
-    const msgCount = Number(await addon.account.getMsgCount());
-    messageCount.innerText = msgCount;
-    if (msgCount === 0) {
+    const { count } = await (await fetch("https://api.scratch.mit.edu/users/World_Languages/messages/count")).json();
```

Comment:
Uh... we're not fetching the currently logged-in user's message count and instead fetching yours?

Reply:
lol 😄

Reply:
It works on my machine !